landing-page/scripts/ic_api.py

```
@@ -333,24 +338,63 @@ def ic_start(project_id: int, body: IcStartRequest, user=Depends(get_current_use
+    covers every path into the encoder rather than only this bridge.
+
+    Note: without ``body.projectId`` this only checks that *some* user is
     logged in, not that they own the project session_id belongs to -- IC's
     own session store is the real authority here, but within mothra's own
     trust model any authenticated user who learns another user's session_id
-    could finalise it. Inconsistent with the rest of this file; revisit if
+    could export it. Inconsistent with the rest of this file; revisit if
     session_id ever becomes guessable/discoverable in practice.
     """
+    if body is not None and body.projectId is not None:
+        with db_cursor() as (con, cur):
```

Ruff (0.16.1)

[warning] Unpacked variable `con` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)